default.env

```diff
@@ -3,10 +3,10 @@
 # https://docs.docker.com/compose/environment-variables/#the-env-file
 # environmental variables for interpolation in docker-compose YAML files
 
-PGRST_JWT_SECRET=<generate via `openssl rand -base64 32`>
+PGRST_JWT_SECRET=<generate via 'openssl rand -base64 32'>
 
 # uncomment to expose postgrest port
-# COMPOSE_FILE=docker-compose.yaml:docker-compose.expose.yaml
+#COMPOSE_FILE=docker-compose.yaml:docker-compose.expose.yaml
```

Comment:
probably want to revert this whitespace only change, as well.  (just to keep the changelog clean)